extensions/undertow/runtime/src/main/java/io/quarkus/undertow/runtime/UndertowDeploymentRecorder.java

```diff
@@ -327,6 +327,8 @@ public void run() {
             @Override
             public void handle(HttpServerRequest event) {
                 VertxHttpExchange exchange = new VertxHttpExchange(event, ALLOCATOR, executorService);
+                httpConfiguration.limits.maxBodySize
```

Comment:
Can this be changed to just a .isPresent() call? Lambdas use a significant amount of metaspace and result in an allocation on every request, so we try and avoid them in the runtime code as much as possible.

Reply:
> Other than those two minor comments looks good

Thanks for the review @stuartwdouglas. New updates pushed. WDYT?